cdap-app-fabric/src/test/java/co/cask/cdap/metadata/LineageAdminTest.java

```diff
@@ -518,4 +510,20 @@ private TransactionExecutorFactory getTxExecFactory() {
   private DatasetFramework getDatasetFramework() {
     return getInjector().getInstance(DatasetFramework.class);
   }
+
+  private static final class NoOpEntityValidator extends EntityValidator {
+    public NoOpEntityValidator() {
+      super(null, null, null, null);
```

Comment:
nit: I don't think we should annotate EntityValidator params with ``@Nullable`` for this, but maybe a comment here saying this is only ok because we are no-oping all functionality in EntityValidator in this class?